reference/polkadot-hub/smart-contracts.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Polkadot Hub Smart Contracts
+description: Learn how Polkadot Hub supports smart contracts through the REVM, a Rust-based Ethereum Virtual Machine compatible runtime.
+categories: Polkadot Protocol
+---
+
+# Smart Contracts on Polkadot Hub
+
+--8<-- 'text/smart-contracts/polkaVM-warning.md'
+
+## Introduction
+
+Polkadot’s relay chain does not support smart contracts directly. Instead, developers deploy contract-based applications on parachains that provide execution environments. Polkadot Hub supports smart contracts through the REVM, a Rust-based Ethereum Virtual Machine compatible runtime.
+
+REVM enables developers to run standard Ethereum bytecode on Polkadot Hub, giving teams a path to migrate existing Solidity contracts while interacting with on-chain assets and other parachains via XCM.
+
+This guide explains how smart contracts are deployed, executed, and integrated on Polkadot Hub using REVM.
+
+<!-- TODO: Can we scrap this?
+ ## Building a Smart Contract
+
+Polkadot Hub supports multiple smart contract environments, giving developers the freedom to choose the workflow and tooling that best fits their project:
+
+- **PolkaVM**: A next-generation virtual machine designed specifically for Polkadot. Built on a high-performance [RISC-V-based register architecture](https://en.wikipedia.org/wiki/RISC-V){target=_blank}, PolkaVM enables fast, scalable execution and is optimized for modern smart contract development.
+- **REVM**: The [REVM backend](https://github.com/bluealloy/revm){target=_blank} brings a full Rust implementation of the EVM to Polkadot Hub. This allows developers to deploy existing Solidity contracts without modification, preserving compatibility with Ethereum tools and libraries.
+
+Each of these environments is fully compatible with Polkadot Hub, giving teams the option to reuse Ethereum code, build with Rust security guarantees, or explore high-performance innovation with PolkaVM.
+
+### PolkaVM Contracts
+
+PolkaVM is Polkadot Hub’s native, high-performance smart contract engine. Instead of emulating EVM bytecode, it executes contracts compiled to a RISC-V instruction set, giving tighter control over execution, metering, and parallelization while staying friendly to Ethereum-style development.
```

Comment:
There is no need to emphasize "tighter execution control" instead you can highlight performance.